src/controllers/authors/create-authors-controller.ts

```diff
@@ -128,14 +126,21 @@ class CreateAuthorsController {
       name: authorNameInputElement.value,
       birthdate: birthdate.toISOString(),
     };
-
+    const toastModalElement = <HTMLDivElement>(
+      document.querySelector("#delete-toast")
```

Comment:
This ID. on all the controllers should be.   create-toast

Reply:
work done